docs/contributing.md

```diff
@@ -94,21 +94,17 @@ When releasing, make sure you respect the tagging:
 - To release pre-release, e.g. `42.1.0-pre.7`, use just `npm publish`.
 - To release any other version, e.g. `42.1.0`, use `npm publish && npm dist-tag add dredd@42.1.0 stable`.
 
-Hopefully this will be automated one day.
+Releasing standard versions is currently automated by [Semantic Release][]. Releasing pre-releases is not automated and needs to be done manually, ideally from a special git branch.
 
 ### Testing
 
-Use `npm run test` or just `npm test` to run all tests. Use `npm run test:bdd`
-to run tests and watch for changes. Dredd uses [Mocha][] as a test framework.
+Use `npm test` to run all tests. Dredd uses [Mocha][] as a test framework.
 It's default options are in the `test/mocha.opts` file.
 
-If you experience a flaky test, you can use `npm run test:stress` in combination
-with `describe.only` to try to replicate the flaky behavior.
-
 ### Integration Tests of Hooks Handlers
 
 Every Pull Request spawns dependent integration builds of hook handlers. Thanks
-to this author of the PR can be sure they did not break hook handler
+to this, author of the PR can be sure they did not break hook handler
```

Comment:
they is a gender-neutral way how to refer to a person in modern English - https://en.wikipedia.org/wiki/Singular_they